cmd/descheduler/app/options/options.go

```diff
@@ -47,15 +47,21 @@ func NewDeschedulerServer() *DeschedulerServer {
 }
 
 // AddFlags adds flags for a specific SchedulerServer to the specified FlagSet
-func (rs *DeschedulerServer) AddFlags(fs *pflag.FlagSet) {
-	fs.DurationVar(&rs.DeschedulingInterval, "descheduling-interval", rs.DeschedulingInterval, "Time interval between two consecutive descheduler executions. Setting this value instructs the descheduler to run in a continuous loop at the interval specified.")
-	fs.StringVar(&rs.KubeconfigFile, "kubeconfig", rs.KubeconfigFile, "File with  kube configuration.")
-	fs.StringVar(&rs.PolicyConfigFile, "policy-config-file", rs.PolicyConfigFile, "File with descheduler policy configuration.")
-	fs.BoolVar(&rs.DryRun, "dry-run", rs.DryRun, "execute descheduler in dry run mode.")
+func (s *DeschedulerServer) AddFlags(fs *pflag.FlagSet) {
+	AddKubeletConfigFlags(fs, &s.DeschedulerConfiguration)
+}
+
+// AddKubeletConfigFlags adds flags for a specific kubeletconfig.KubeletConfiguration to the specified FlagSet
+func AddKubeletConfigFlags(fs *pflag.FlagSet, c *componentconfig.DeschedulerConfiguration) {
+	fs.DurationVar(&c.DeschedulingInterval, "descheduling-interval", c.DeschedulingInterval, "Time interval between two consecutive descheduler executions. Setting this value instructs the descheduler to run in a continuous loop at the interval specified.")
+	fs.StringVar(&c.KubeconfigFile, "kubeconfig", c.KubeconfigFile, "File with  kube configuration.")
+	fs.StringVar(&c.PolicyConfigFile, "policy-config-file", c.PolicyConfigFile, "File with descheduler policy configuration.")
+	fs.StringVar(&c.Logging.Format, "logging-format", c.Logging.Format, `Sets the log format. Permitted formats: "text", "json".\nNon-default formats don't honor these flags: -add_dir_header, --alsologtostderr, --log_backtrace_at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --skip_headers, --skip_log_headers, --stderrthreshold, --log-flush-frequency.\nNon-default choices are currently alpha and subject to change without warning.`)
```

Comment:
```suggestion
	fs.StringVar(&c.Logging.Format, "logging-format", c.Logging.Format, `Sets the log format. Permitted formats: "text", "json".\nNon-default formats don't honor these flags: -add_dir_header, --alsologtostderr, --log-backtrace-at, --log-dir, --log-file, --log-file-max-size, --logtostderr, --skip-headers, --skip-log-headers, --stderrthreshold, --log-flush-frequency.\nNon-default choices are currently alpha and subject to change without warning.`)
```

Reply:
I've removed it altogether, as I found `k8.io/component-base/logs` package has a method for it.

https://github.com/kubernetes/component-base/blob/c985e940ef8ffde448c517479bf260be98ce596b/logs/options.go#L84-L91